packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -1094,10 +1086,10 @@ test.describe.serial('Invalidation', () => {
 
 	test('$page.url can safely be mutated', async ({ page }) => {
 		await page.goto('/load/mutated-url?q=initial');
-		expect(await page.textContent('h1')).toBe('initial');
+		expect(page.getByText('initial')).toBeVisible();
```

Comment:
surely this needs an `await` otherwise it's not testing anything at all?

```suggestion
		await expect(page.getByText('initial')).toBeVisible();
```